README.md

```diff
@@ -3762,3 +4352,412 @@ return Http::withOptions([
+                    ->group(base_path('routes/local.php'));
+            }
+        });
+    }
+}
+```
+
+Tip given by [@Philo01](https://twitter.com/Philo01/status/1466068376330153984)
+
+### You can send e-mails to a custom log file
+In Laravel you can send e-mails to a custom log file.
+
+You can set your environment variables like this:
+
+```
+MAIL_MAILER=log
+MAIL_LOG_CHANNEL=mail
+```
+
+And also configure your log channel:
+```php
+'mail' => [
+    'driver' => 'single',
+    'path' => storage_path('logs/mails.log'),
+    'level' => env('LOG_LEVEL', 'debug'),
+],
+```
+
+Now you have all your e-mails in /logs/mails.log<br>
+It's a good use case to quickly test your mails.
+
+Tip given by [@mmartin_joo](https://twitter.com/mmartin_joo/status/1466362508571131904)
+
+### Markdown made easy
+Laravel provides an interface to convert markdown in HTML out of the box, without the need to install new composer packages.
+```php
+$html = Str::markdown('# Changelogfy')
+```
+
+Output:
+```
+<h1>Changelogfy</h1>
+```
+
+Tip given by [@paulocastellano](https://twitter.com/paulocastellano/status/1467478502400315394)
+
+### Simplify if on a request with whenFilled() helper
+We often write if statements to check if a value is present on a request or not.<br>
+You can simplify it with the `whenFilled()` helper.
+
+```php
+public function store(Request $request)
+{
+    $request->whenFilled('status', function (string $status)) {
+        // Do something amazing with the status here!
+    }, function () {
+        // This it called when status is not filled
+    });
+}
+```
+
+Tip given by [@mmartin_joo](https://twitter.com/mmartin_joo/status/1467886802711293959)
+
+### Use observers if you need to listen to many events
+If you need listening to many events from the same model in #laravel, a great option is to use #observers.<br>
+In this example, I refresh the user cache every update.
+
+```php
+class UserObserver
+{
+    public function updated(User $user)
+    {
+        UpdateUserCache::dispatch($user);
+    }
+}
+```
+
+Tip given by [@paulocastellano](https://twitter.com/paulocastellano/status/1468277233332756481)
+
+### Make use of the 'abort', 'abort_if' and 'abort_unless' helpers
+Make use of the `abort`, `abort_if` and `abort_unless` helpers to throw an HTTP exception and halt execution at any layer of your application.
+```php
+class DeletePodcastAction
+{
+    public function execure(User $user, Podcast $podcast)
+    {
+        if (! $podcast->owner($user)) {
+            abort(403, 'You do not own this podcast');
+        }
+    }
+}
+```
+
+Laravel also includes the usefull `abort_if` and `abort_unless` helpers, which we can use to simplify any conditionals
+```php
+abort_if($podcast->isLive(), 403, 'The podcast is currently streaming');
+
+abort_unless($podcast->owner($user), 403, 'You do not own this podcast');
+```
+
+Tip given by [@mattkingshott](https://twitter.com/mattkingshott/status/1470423275658481673)
+
+### Laravel's chain magic
```

Comment:
@krekas I don't think this tip is worth adding. I just personally don't have the case where I would use this one, and it even makes the code LONGER.